crypto/fipsmodule/bn/bn_assert_test.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+//
+
+#include <openssl/bn.h>
+#include <openssl/rand.h>
+#include "./internal.h"
+
+#include <gtest/gtest.h>
+
+TEST(BNAssertTest, Assert_fits_in_bytes_large) {
+// TODO: Update Android test harness
+#if !defined(NDEBUG) && !defined(OPENSSL_ANDROID)
```

Comment:
Did you consider surrounding the entire file with the excluding flags? Or maybe exclude it from a cmake build?

Reply:
I think it's better for the test to still be reported/runnable. We've made similar decisions elsewhere, for example the ["dynamic_loading_test"](https://github.com/aws/aws-lc/blob/main/crypto/dynamic_loading_test.c#L115) just outputs "PASS" when the CI runs the test on Windows.